source/community/reactnative/src/components/actions/select-Action.js

```diff
@@ -58,7 +57,7 @@ export class SelectAction extends React.Component {
 	}
 
 	render() {
-		if (HostConfigManager.getHostConfig().supportsInteractivity === false) {
+		if (!this.props.configManager.hostConfig.supportsInteractivity) {
```

Comment:
Please confirm supportsInteractivity is not true by default. In such case, this would fail.

Reply:
I think **`supportsInteractivity`** should be true by default, only then the `SelectAction` component will get rendered. If it is false then we won't be using the select action and it will return `null`.

Reply:
Ok. As you said, i just checked supportsInteractivity is set to true on undefined case. This looks good to me now.